ai-platform-unified/notebooks/official/pipelines/pipelines_intro_kfp.ipynb

```diff
@@ -0,0 +1,1109 @@
+{
```

Comment:
<p>arg -&gt; argument</p><p> </p><p>"Note that you can pass a <code>parameter_values</code> dict that specifies the pipeline input parameters you want to use."</p><p> </p><p>I found this confusing. Is it optional or required?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
<p>It is optional as long as all pipeline input params have defaults that you want to use. </p>